pipes/WDL/workflows/scaffold_and_refine_multitaxa.wdl

```diff
@@ -100,14 +113,10 @@ workflow scaffold_and_refine_multitaxa {
     }
 
     output {
-        Array[Map[String,String]] assembly_stats_by_taxon = stats_by_taxon
-        File   assembly_stats_by_taxon_tsv = concatenate.combined
-
-        Int    num_read_groups                       = refine.num_read_groups[0]
-        Int    num_libraries                         = refine.num_libraries[0]
+        Array[Map[String,String]] assembly_stats_by_taxon  = stats_by_taxon
```

Comment:
Any reason we can't make this type `Map[String, Map[String,String] ]`, where the outer map `String` keys are the `taxid` or  `tax_name` values? (for picking out values for a given taxon in downstream analyses)

Reply:
Mostly just because of how we construct it (see the scatter in the WDL above), and that WDL 1.0 lacks a lot of the basic methods for navigating `Map`s and converting back and forth with `Array`s.